File: go-sdk/pkg/models/operations/deleteabranchpassword.go
package operations

type DeleteABranchPasswordPathParams struct {
	Branch       string `pathParam:"style=simple,explode=false,name=branch"`
	Database     string `pathParam:"style=simple,explode=false,name=database"`
	ID           string `pathParam:"style=simple,explode=false,name=id"`
	Organization string `pathParam:"style=simple,explode=false,name=organization"`
}

type DeleteABranchPasswordRequest struct {
	PathParams DeleteABranchPasswordPathParams
}

type DeleteABranchPasswordResponse struct {
	ContentType string
	StatusCode  int
}

File: ts-sdk/src/sdk/models/operations/deleteabranchpassword.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";
import { Type } from "class-transformer";


export class DeleteABranchPasswordPathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=branch" })
  branch: string;

  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=database" })
  database: string;

  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=id" })
  id: string;

  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=organization" })
  organization: string;
}

export class DeleteABranchPasswordRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: DeleteABranchPasswordPathParams;
}

export class DeleteABranchPasswordResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;
}